src/index.html

```diff
@@ -7,6 +7,50 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="sparkline">
+      <div class="sparkline__day">mon</div>
+      <div class="sparkline__day">tue</div>
+      <div class="sparkline__day">wed</div>
+      <div class="sparkline__day">thu</div>
+      <div class="sparkline__day">fri</div>
+      <div class="sparkline__day">sat</div>
+      <div class="sparkline__day">sun</div>
+    </div>
+
+    <div class="calendar calendar--month-length_31">
+      <div class="calendar__field">
+        <div class="calendar__day calendar__day--start-day_fri"></div>
```

Comment:
It's a `calendar` block modifier, move it to the calendar classes. And rename following BEM naming convention for complex modifiers.

Reply:
I already renamed modifier for first day position, and actually it`s modifier exactly for the first day in month, month could change for any another in this case, so we had to change only first day position. 

In everything else i agree with you fully

Reply:
I understand that you already implemented start-day modifier this way, but according to the task, `start-day` should be modifier for the `calendar`, not for `calendar__day`.